sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v2/checks.sql

```diff
@@ -1,2 +1,12 @@
-ASSERT((SELECT COUNT(*) FROM `{{project_id}}.{{dataset_id}}.{{table_name}}` WHERE download_date = @download_date) > 250000)
-AS 'ETL Data Check Failed: Table {{project_id}}.{{dataset_id}}.{{table_name}} contains less than 250,000 rows for date: {{ download_date }}.'
\ No newline at end of file
+ASSERT(
```

Comment:
Format changes only.

Reply:
Curious - are these format changes automated or manual?

Reply:
They're automated in the CI. Locally I run them using ./bqetl format.
E.g. to check all the query files `./bqetl format $(git ls-tree -d HEAD --name-only)`

Reply:
Interesting that it picked up the checks.sql not just query.sql files.  One less thing to do to cover the checks.sql files.